File: API/Python/Hosting/listImage.py
#!/usr/bin/python3

import xmlrpc.client
api = xmlrpc.client.ServerProxy('https://rpc.gandi.net/xmlrpc/')

apikey = '<api_key>'

images = api.hosting.image.list(apikey, {'datacenter_id': 1})
debian_images = [x for x in images if x['label'].lower().startswith('debian')]

print(debian_images)

File: API/Php/Hosting/listImage.php
<?php
require_once 'XML/RPC2/Client.php';

$apikey = '<api_key>';

// Image available list
$api_hosting = XML_RPC2_Client::create(
    'https://rpc.gandi.net/xmlrpc/',
    array( 'prefix' => 'hosting.image.', 'sslverify' => False )
);

$result = $api_hosting->list($apikey);
print_r($result);

?>
